bpf/process/bpf_process_event.h

```diff
@@ -371,14 +376,16 @@ getcwd(struct msg_process *curr, __u32 offset, __u32 proc_pid)
 	struct fs_struct *fs;
 	int flags = 0, size;
 	char *buffer;
+	__u64 ino = 0;
+	__u32 dev = 0;
 
 	probe_read(&fs, sizeof(fs), _(&task->fs));
 	if (!fs) {
 		curr->flags |= EVENT_ERROR_CWD;
 		return 0;
 	}
 
-	buffer = d_path_local(_(&fs->pwd), &size, &flags);
+	buffer = d_path_local(_(&fs->pwd), &size, &flags, &ino, &dev);
```

Comment:
just pass null, null in here and check ino/dev != null in path_with_deleted,
no need for extra assignment if we have no use for it